src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/AddIndexCommand.java

```diff
@@ -8,7 +8,8 @@
 import jakarta.validation.constraints.Size;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-@Schema(description = "Command that creates an index for a column in a table.")
+// TODO, hide table feature detail before it goes public
```

Comment:
idea: maybe link back to issue for re-enabling these annotations (may need to create one if it doesn't exist)?

Reply:
No problem.